--- ../bash-4.4/parse.y	Sun Sep 11 10:31:46 2016
+++ parse.y	Fri Dec 14 10:03:10 2018
@@ -35,6 +35,7 @@
 #endif
 
 #include <stdio.h>
+#include <errno.h>
 #include "chartypes.h"
 #include <signal.h>
 
@@ -132,7 +133,9 @@
 extern int bash_input_fd_changed;
 #endif
 
+#ifndef errno
 extern int errno;
+#endif
 /* **************************************************************** */
 /*								    */
 /*		    "Forward" declarations			    */
